libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc

```diff
@@ -118,9 +118,9 @@ mode.
 `namespace`:: (Optional) Select the namespace from which to collect the
 metadata. If it is not set, the processor collects metadata from all namespaces.
 It is unset by default.
-`add_resource_metadata`:: (Optional) Specify labels and annotations filters for the extra metadata coming from Node and Namespace.
+`add_resource_metadata`:: (Optional) Specify labels and annotations filters for the extra metadata coming from Node and namespace.
```

Comment:
Why is Node capitalized? It should be lowercase, no? See the [K8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/).